calyx-py/test/correctness/arbiter_6.expect

```diff
@@ -0,0 +1,47 @@
+{
+  "cycles": 27,
+  "memories": {
+    "A": [
+      1,
+      2,
+      3,
+      4
+    ],
+    "B": [
+      5,
+      6,
+      7,
+      8
+    ],
+    "C": [
+      9,
+      10,
+      11,
+      12
+    ],
+    "D": [
+      13,
+      14,
+      15,
+      16
+    ],
+    "E": [
+      17,
+      18,
+      19,
+      20
+    ],
+    "F": [
+      21,
+      22,
+      23,
+      24
+    ],
+    "out2": [
+      0
+    ],
+    "out3": [
+      0
```

Comment:
@anshumanmohan are these values correct?

Reply:
Nope, something is up. Yesterday I was getting 24 and 24, i.e. showing how to access the last value of each arbitered cell. Leave it to me, I'll bike in to fix it before the meeting